### PR TITLE
On 961 api put existing contact

### PR DESCRIPTION
**Pull request by MarkFCain**
Another PR against the branch before.

### CODE REVIEW

File: src/SFA.DAS.AssessorService.Api.Types/Models/Register/UpdateEpaOrganisationContactRequest.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using MediatR;
+
+namespace SFA.DAS.AssessorService.Api.Types.Models.Register
+{
+    public class UpdateEpaOrganisationContactRequest : IRequest<string>
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Api.Types/Models/Register/UpdateEpaOrganisationContactRequest.cs]: Just wondering if this be better if it returned a Contact object or an UpdateEpaOrganisationContactResponse than a string

File: src/SFA.DAS.AssessorService.Data/RegisterValidationRepository.cs [hunk 2]
```diff
@@ -143,5 +143,31 @@ public async Task<bool> EmailAlreadyPresentInAnotherOrganisation(string email, s
...
+
+        public async Task<bool> ContactExists(string contactId)
+        {
+            using (var connection = new SqlConnection(_configuration.SqlConnectionString))
+            {
+                if (connection.State != ConnectionState.Open)
+                    await connection.OpenAsync();
+                const string sqlToCheckExists =
+                    "select CASE count(0) WHEN 0 THEN 0 else 1 end result FROM [Contacts] " +
+                    "WHERE convert(varchar(50),id)  = @contactId";
```
Comment (martin-hornsby) [inline src/SFA.DAS.AssessorService.Data/RegisterValidationRepository.cs]: Could this be done using GUID than string?

File: src/SFA.DAS.AssessorService.Api.Types/Models/Register/UpdateEpaOrganisationContactRequest.cs (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.AssessorService.Api.Types/Models/Register/UpdateEpaOrganisationContactRequest.cs]: I don't need the contact returned, just the id of the contact to work with the MVC stateless flow.  There's a separate endpoint to get the contact from the id returned, if needed.

**Comment by MarkFCain** [comment]
Closed as will be merged as part of ON-966

**Pull request closed without merging**